Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs

```diff
@@ -541,14 +554,16 @@ public void TestWithoutTor()
                         }
                     }
 
-                    Thread.Sleep(20000);
+                    Thread.Sleep(5000);
```

Comment:
Be careful setting this too low, there are delays in the runtime itself (although they are artificially decreased for regtest). It also takes time to perform the state machine updates in the crypto-heavy phases when multiple cycles are running concurrently. I found that even 10 seconds was marginally stable. 20 seconds slows down the test but at least seems to be a workable setting.

Reply:
Restored to original value